src/main/java/mate/academy/service/Main.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.service;
+
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+public class Main {
+    private static final Injector injector
+            = Injector.getInstance("mate.academy.dao");
+
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book();
+        book.setTitle("Java Programming");
+        book.setPrice(BigDecimal.valueOf(29.99));
+
+        bookDao.create(book);
+        System.out.println("The book added: " + book);
+
+        Book foundBook = bookDao.findById(book.getId()).orElse(null);
+        if (foundBook == null) {
```

Comment:
Using `Optional.orElse(null)` is not the best practice. Consider using `ifPresent` to handle the presence of a value or `orElseThrow` to handle the absence more explicitly.